design/src/components/Glogg/Glogg.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Champagne from "../../assets/img/champagne.jpg";
+import Pear from "../../assets/img/margaret.jpg";
+import Berries from "../../assets/img/berries.jpg";
+import Saffron from "../../assets/img/saffron.jpg";
+
+const Glogg = () => {
+    return (
+        <div className="glogg-container">
+            <div className="glogg-row1">
+                <h1>Årets glöggsmaker</h1>
+            </div>
+
+            <div className="glogg-row2">
+                <div className="glogg-ribbon">
+                    <p className="glogg-ribbon-text">Passar till chokladälskare och miljömedvetna</p>
+                </div>
+                <p className="glogg-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            </div>
+
+            <div className="glogg-row3">
+                <div className="glogg">
+                    <img src={Champagne} alt="Glögg Champagne" width="300" height="200"/>
+                    <div className="gloggText">
```

Comment:
Kanske vi ska hålla oss till att använda oss av bindestreck som klassnamn och inte blanda in camel case?

Reply:
Bra poänger @josefinelofgren !